ui/entry-perf.js

```diff
@@ -22,9 +22,6 @@ import './css/treeherder-loading-overlay.css';
 // Bootstrap the Angular modules against which everything will be registered
 import './js/perf';
 
-// React UI
-import './shared/auth/Login';
```

Comment:
This is being removed since it's a leftover from:
https://github.com/mozilla/treeherder/pull/4167#pullrequestreview-167564835